### DISCUSSION TITLE
Google Adsense Script

### DISCUSSION THREAD

**Discussion by orcunkobal** [remix-run/remix#5825] [Q&A] [answered]
Hi everyone,

I wan't to add google adsense script in root.jsx like this;

```
return (
  <html lang="en">
      <head>
            <Meta />
            <Links />
            <script src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js?client=ca-pub-8679629074408273" crossOrigin="anonymous" async></script>
      </head>
      <body>
            <ScrollRestoration />
            <ExternalScripts />
            <Scripts />
            <LiveReload />
      </body>
  </html>)
```

But this script want's to add some other adsense dependencies such as scripts and iframe inside of the header and body, the problem is start here...  Actually it tries to add  but quickly removing from the DOM by remix and the below error appears. 
<img width="1308" alt="Screenshot 2023-03-16 at 14 03 59" src="https://user-images.githubusercontent.com/6178923/225598607-5047314c-1522-489c-8243-f26f9eada131.png">

Anyone add adsense to their remix project? Please give an advices :)

Remix version is : ^1.13.0
React version is : ^18.2.0

**Comment by joeykamsteeg** [comment]
In production build I do get `Minified React error #418 #425 #423`

**Comment by joeykamsteeg** [comment, marked as answer]
I guess this is related to #5244

**Comment by orcunkobal** [comment]
Yes, I fixed the issue with #5244 

thanks :)
